### PR TITLE
[ES-5952] send batch events

### PR DESCRIPTION
**Pull request by srfication**

### CODE REVIEW

File: js/visearch.js [hunk 1]
```diff
@@ -86,6 +90,25 @@ const RESULT_LOAD = 'result_load';
...
+      tracker = getTracker();
+
+      if (tracker && tracker.validateEvents(events, failure)) {
+        const batchId = tracker.generateUUID();
+
+        events.forEach((event) => {
+          event.v = SDK_VERSION;
+          event.sdk = SDK;
+
+          if (action.toLowerCase() === 'transaction' && !event.transId) {
```
Comment (Sisyphus25) [inline js/visearch.js:103]: This means that all scripts (widget/deploy script/banner) should also support non case sensitive transaction events right?

Comment (srfication) [reply js/visearch.js:103]: ideally yes.

Comment (srfication) [reply js/visearch.js:103]: same goes for add_to_cart as well

Comment (srfication) [reply js/visearch.js:103]: in our data pipeline we will also auto convert all actions to lowercase

Comment (Sisyphus25) [reply js/visearch.js:103]: Does this also apply for `add_to_cart`?

**Pull request merged**